package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "to-do-list-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
- [ ] Kindly consider adding a script for running your project, so any one can run your project, By adding the following line `"start": "webpack serve --open",`